Decred Wallet/Utils/LocalizedStrings.swift

```diff
@@ -0,0 +1,401 @@
+    static let accounts = NSLocalizedString("accounts", comment: "")
+    static let security = NSLocalizedString("security", comment: "")
+    static let settings = NSLocalizedString("settings", comment: "")
+    static let help = NSLocalizedString("help", comment: "")
+    
+    /* NavigationMenu / Sync progress */
+    static let cannotSyncWithoutNetworkConnection = NSLocalizedString("cannotSyncWithoutNetworkConnection", comment: "")
+    static let internetConnectionRequired = NSLocalizedString("internetConnectionRequired", comment: "")
+    static let connectToWiFiToSync = NSLocalizedString("connectToWiFiToSync", comment: "")
+    static let restartingSync = NSLocalizedString("restartingSync", comment: "")
+    static let connectingToPeers = NSLocalizedString("connectingToPeers", comment: "")
+    static let syncedWith =  NSLocalizedString("syncedWith", comment: "")
+    static let fetchingBlockHeaders = NSLocalizedString("fetchingBlockHeaders", comment: "")
+    static let blocksBehind = NSLocalizedString("blocksBehind", comment: "")
+    static let bestBlockAgeAgo = NSLocalizedString("bestBlockAgeAgo", comment: "")
+    static let discoveringUsedAddresses = NSLocalizedString("discoveringUsedAddresses", comment: "")
+    static let generalSyncProgressCompletedleft = NSLocalizedString("generalSyncProgressCompletedleft", comment: "")
+    static let scanningBlocks = NSLocalizedString("scanningBlocks", comment: "")
+    static let rescanProgress = NSLocalizedString("rescanProgress", comment: "")
+    static let syncCanceled = NSLocalizedString("syncCanceled", comment: "")
+    static let syncError = NSLocalizedString("syncError", comment: "")
+    static let latestBlock = NSLocalizedString("latestBlock", comment: "")
+    static let now = NSLocalizedString("now", comment: "")
+    static let secondsAgo = NSLocalizedString("secondsAgo", comment: "")
+    static let minAgo = NSLocalizedString("minAgo", comment: "")
+    static let hrsAgo = NSLocalizedString("hrsAgo", comment: "")
+    static let daysAgo = NSLocalizedString("daysAgo", comment: "")
+    static let weeksAgo = NSLocalizedString("weeksAgo", comment: "")
+    static let monthsAgo = NSLocalizedString("monthsAgo", comment: "")
+    static let yearsAgo = NSLocalizedString("yearsAgo", comment: "")
+    static let totalBalance = NSLocalizedString("totalBalance", comment: "")
+    static let restartingSynchronization = NSLocalizedString("restartingSynchronization", comment: "")
+    static let startingSynchronization = NSLocalizedString("startingSynchronization", comment: "")
+    
+    /* Sync progress view */
+    static let loading = NSLocalizedString("loading", comment: "")
+    static let numberOfConnectedPeers = NSLocalizedString("numberOfConnectedPeers", comment: "")
+    static let numberOfConnectedPeer = NSLocalizedString("numberOfConnectedPeer", comment: "")
+    static let syncingWithOnNet = NSLocalizedString("syncingWithOnNet", comment: "")
```

Comment:
I wondered for a minute what `syncingWithOnNet` meant until I searched around to see that it refers to the string `"Syncing with %@ on %@."`
`syncingWithPeersOnNetwork` would be a more descriptive variable name.